packages/replay/src/auth.ts

```diff
@@ -263,3 +282,115 @@ export async function maybeAuthenticateUser(options: Options = {}) {
     return false;
   }
 }
+
+async function getAuthInfo(key: string): Promise<string> {
```

Comment:
It could be nice to add somewhere a definition of the `authInfo` since it's just a string rn. This function wouldn't be the right place to do that since when we are reading it that information is quite close - I was wondering about it when reading some other parts of the code. But then... where would a good place for this be? I don't know tbh ;p 

Maybe consolidating all of this authInfo-related stuff into a module/class/object would be a good thing? Then all of the operations related to it would be close to each other and a piece of comment explaining what an authinfo is could live above it.

A piece of context why I was looking for it in the first place: I was wondering what `auth.json` and `authInfo.json` are about and how they relate to each other.

I know that everything here is already in `auth.json` file so maybe I'm just splitting hairs here as I'm just learning how things work. Feel free to treat this comment as a nit and ignore it.

Reply:
For an explanation about authInfo, I see your point but right now I think it's good enough if I explain what it is right and why we need it right above `initLDContextFromApiKey`. We can look into splitting out in separate class but I don't want prematurely do it.


>  I was wondering what auth.json and authInfo.json are about and how they relate to each other

`auth.json` has a user token but not the user's ID. We could potentially save everything in a single file but I felt authInfo is more like a cache that we should be able to remove if we need it without any change in functionality and that's why kept it separate. We can revisit this if we want to consolidate these two.